tests/agents/test_opencode_mcp_integration.py

```
@@ -0,0 +1,397 @@
+            "getPackageDependencies": self.toolkit.read_packages,
+            "getControlFlowGraph": self.toolkit.read_cfg,
+            "getMethodInvocations": self.toolkit.read_method_invocations,
+            "readDocs": self.toolkit.read_docs,
+            "readExternalDeps": self.toolkit.external_deps,
+        }
+        if tool_name not in tool_map:
+            return {"content": [{"type": "text", "text": f"Error: Unknown tool '{tool_name}'"}], "isError": True}
+        try:
+            result = tool_map[tool_name]._run(**args)
+            return {"content": [{"type": "text", "text": str(result)}], "isError": False}
+        except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)